contracts/queue/src/lib.rs

```diff
@@ -1,8 +1,8 @@
 pub mod contract;
 
-/** Below we expose wasm exports **/
 #[cfg(target_arch = "wasm32")]
-pub use cosmwasm_std::{allocate, deallocate};
+#[allow(unused_imports)]
+use cosmwasm_std; // Pull in cosmwasm_std's C externs (cosmwasm_vm_version_1, allocate, deallocate)
```

Comment:
SInce `cosmwasm_std` is surely imported by `contract.rs`, (and also by `mod wasm` below) isn't that enough?
What happens if you remove this line?

Reply:
We can remove it as long at it is imported somewhere in the contract. I kept it here to explain how cosmwasm_vm_version_1, allocate, deallocate work.

Reply:
It is imported a few lines below in `mod wasm`. Maybe just move the comment there?

Reply:
Like this? https://github.com/CosmWasm/cosmwasm/pull/209/commits/2a75ee2ff9713cac783979f2cfdcb1cf7d4ffac3

Reply:
Yes, looks good